### DISCUSSION TITLE
alist更新得好频繁，能列出每次更新内容吗？2.5.0更新太棒了！

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1061] [General]
每次更新不知道更新了啥内容，能列一个更新记录吗？
2.5.0更新了aria2功能，太棒了，这样网盘同步变得非常轻松了。

**Comment by anwen-anyi** [comment]
https://alist-doc.nn.ci/docs/changelog/v2.5.x

**Reply by fzlychee2019** [reply]
感谢官方辛勤的更新，太赞了！
